core/src/main/java/apoc/trigger/TriggerNewProcedures.java

```diff
@@ -48,7 +49,7 @@ private void checkInSystem() {
         }
     }
 
-    private void checkTargetDatabase(String databaseName) {
+    private void checkTargetDatabase(String databaseName, Transaction tx) {
```

Comment:
You don't need the transaction here I think? Because it's the same one from the class